creator/templates/rsa_jwk.j2

```diff
@@ -0,0 +1,10 @@
+      {
+         "id": "{{ issuer }}#JWK2020-RSA-key#{{ number }}",
+         "type": "JsonWebKey2020",
+         "controller": "{{ issuer }}",
+         "publicKeyJwk": {
+            "kty": "RSA",
+            "n": "{{ jwk.n }}",
+            "e": "{{ jwk.e }}"
+         }
```

Comment:
Should support `x5u` and `x5c` optionally. See the comment on the other template.

Reply:
You are right.